readme.md

```diff
@@ -643,6 +643,7 @@
 - [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
 - [Prompt Injection](https://github.com/FonduAI/awesome-prompt-injection#readme) - A type of vulnerability that specifically targets machine learning models.
 - [Detection Engineering](https://github.com/infosecB/awesome-detection-engineering#readme) - Design, build, and operate detective cybersecurity controls.
+- [Red-Teaming LLMs](https://github.com/dapurv5/awesome-red-teaming-llms#readme) - Systematizing Large Language Model Security: Attack Taxonomy, Defense Framework, and Red-Teaming Protocols
```

Comment:
Missing `.` in the of the sentence:
<img width="791" alt="image" src="https://github.com/user-attachments/assets/120843d6-c5e2-4210-96e6-8d5dc96d3fec" />

```
Systematizing Large Language Model Security: Attack Taxonomy, Defense Framework, and Red-Teaming Protocols.
```